src/frontend/src/btc/components/send/BtcSendForm.svelte

```diff
@@ -14,6 +17,16 @@
 
 	let amountError: BtcAmountAssertionError | undefined;
 	let invalidDestination: boolean;
+
+	$: {
+		if (nonNullish($authIdentity) && nonNullish(networkId)) {
```

Comment:
- Can we initiate the load in the `onMount` hook? 
- Should we do anything if `$authIdentity` is not defined?
- Can you add a comment that we on purpose does not await the response?

Reply:
> * Can we initiate the load in the `onMount` hook?  Or `networkId` can be manually selected by the user?

Not network id, but also the source. I guess that not at the moment because we don't support subaccounts, but if we support subaccounts in the future is better if this is triggered when `source` changes.

Does that make sense? I can put it in the `onMount` as well, not a strong opinion. Just sharing why I did so. I also put it in `onMount` first but then I thought of changing the `source`.

> * Should we do anything if `$authIdentity` is not defined?

This is just an edge case. An anonymous user can't reach this component. If they do, we also don't want to trigger the call. But I don't see a reason to do anything special either.

> * Can you add a comment that we on purpose does not await the response?

Added!

Reply:
> Not network id...

Fair point. On the other side, if we support subaccounts, then we might need some debouncing as we might not want to trigger lots of call update if the users repeatdly switch between accounts, so we would have to comeback to this too.

For those reason, kind of feel like `onMount` would be nice for now as it would express the fact that we indeed only load it once.

> This is just an edge case

Given that we ignore the error anyway, maybe the service then can accept an `OptionIdentity` and return `{success: false}` if not set that way you don't have to handle the edge case in the component? If agree, we can add a TODO.

Reply:
Done!

I also allowed `networkId` to be `undefined` in the service and I handled the scenario there.